### DISCUSSION TITLE
[2445690] 失落城堡2 {本体2024.8.22更新}

### DISCUSSION THREAD

**Discussion by E1than9** [pjy612/SteamManifestCache#775] [Share]
### Steam 游戏ID

2445690

### 密钥列表

```hash
2445691;58aa213ce335109a11490383c49a406f0bb7a5c68f2844e33e7883c4e70dc1b3
```

### 清单附件

[2445691.zip](https://github.com/user-attachments/files/16711617/2445691.zip)

### 备注

email:[m17751873044@163.com]

**Comment by pjy612** [comment]
* #744

**Reply by E1than9** [reply]
Sorry, I didn't notice
